climada/hazard/trop_cyclone.py

```diff
@@ -1128,29 +1112,27 @@ def compute_angular_windspeeds(si_track, d_centr, close_centr_msk, model, cyclos
     return result
 
 def get_close_centroids(
-    t_lat: np.ndarray,
-    t_lon: np.ndarray,
+    si_track: xr.Dataset,
     centroids: np.ndarray,
     buffer_km: float,
     metric: str = "equirect",
 ) -> np.ndarray:
     """Check whether centroids lay within a buffer around track positions
 
-    The longitudinal coordinates are assumed to be normalized around a central longitude. This
-    makes sure that the buffered bounding box around the track doesn't cross the antimeridian.
-
-    The only hypothetical problem occurs when a TC track is travelling so far in longitude that
-    adding a buffer exceeds 360 degrees (i.e. crosses the antimeridian).
-    Of course, this case is physically impossible.
+    Note that, hypothetically, a problem occurs when a TC track is travelling so far in longitude
+    that adding a buffer exceeds 360 degrees (i.e. crosses the antimeridian), which is physically
+    impossible, but might happen with synthetical or test data.
 
     Parameters
     ----------
-    t_lat : np.ndarray of shape (npositions,)
-        Latitudinal coordinates of track positions.
-    t_lon : np.ndarray of shape (npositions,)
-        Longitudinal coordinates of track positions, normalized around a central longitude.
+    si_track : xr.Dataset
+        Track information as returned by `tctrack_to_si`. Hence, longitudinal coordinates are
```

Comment:
docstring formatting
```suggestion
        Track information as returned by ``tctrack_to_si``. Hence, longitudinal coordinates are
```